diff --git a/node_modules/@react-native-firebase/app-check/ios/RNFBAppCheck/RNFBAppCheckModule.h b/node_modules/@react-native-firebase/app-check/ios/RNFBAppCheck/RNFBAppCheckModule.h
index 999f403..0488580 100644
--- a/node_modules/@react-native-firebase/app-check/ios/RNFBAppCheck/RNFBAppCheckModule.h
+++ b/node_modules/@react-native-firebase/app-check/ios/RNFBAppCheck/RNFBAppCheckModule.h
@@ -15,9 +15,8 @@
  *
  */
 
-#import <Foundation/Foundation.h>
-
 #import <React/RCTBridgeModule.h>
+#import <Foundation/Foundation.h>
 
 #import "RNFBAppCheckProviderFactory.h"
 
diff --git a/node_modules/@react-native-firebase/app-check/ios/RNFBAppCheck/RNFBAppCheckModule.m b/node_modules/@react-native-firebase/app-check/ios/RNFBAppCheck/RNFBAppCheckModule.m
index f112167..569de4c 100644
--- a/node_modules/@react-native-firebase/app-check/ios/RNFBAppCheck/RNFBAppCheckModule.m
+++ b/node_modules/@react-native-firebase/app-check/ios/RNFBAppCheck/RNFBAppCheckModule.m
@@ -15,6 +15,7 @@
  *
  */
 
+#import <React/RCTBridgeModule.h>
 #import <Firebase/Firebase.h>
 #import <FirebaseAppCheck/FIRAppCheck.h>
 
